client/landing/gutenboarding/components/plans/plans-button/style.scss

```diff
@@ -1,7 +1,12 @@
 @import 'assets/stylesheets/gutenberg-base-styles';
+@import '../variables.scss';
```

Comment:
Shouldn't need this import — the CSS color variables are globals (compared to sass variables that we need to import)